### PR TITLE
Replaced the calls to console.log() with writeSync() 

### PR DESCRIPTION
**Pull request by kruffin**
Replaced the calls to console.log() with writeSync() to prevent commands from lingering on the buffer and causing a timeout.

### CODE REVIEW

File: ants/dist/starter_bots/javascript/Ants.js [hunk 1]
```diff
@@ -109,10 +111,11 @@ exports.ants = {
 	'finishTurn': function() {
 		for (var i = 0, len = this.orders.length; i < len; ++i) {
 			var order = this.orders[i];
-			console.log('o '+order.row+' '+order.col+' '+order.direction);
```
Comment (kruffin) [inline ants/dist/starter_bots/javascript/Ants.js:112]: From http://nodejs.org/docs/v0.4.8/api/all.html#process.stdout write is asynchronous and is not desired in this time sensitive application.

console.log = function (d) {
  process.stdout.write(d + '\n');
};

File: ants/dist/starter_bots/javascript/Ants.js [hunk 2]
```diff
@@ -109,10 +111,11 @@ exports.ants = {
...
+			fs.writeSync(process.stdout.fd, 'o '+order.row+' '+order.col+' '+order.direction+'\n');
```
Comment (kruffin) [inline ants/dist/starter_bots/javascript/Ants.js:114]: This is the way to force writing from: http://stackoverflow.com/questions/6471004/how-can-i-write-blocking-in-stdout-with-node-js

**Comment by Janzert** [comment]
Thanks for the patch.

**Comment by kruffin** [comment]
No problem. Just checked this on my Mac and it looks like the fs.fsyncSync() method isn't supported there (tested on Windows before). If you can test it out on linux that would be great. I have a feeling that line will need to be removed though (http://groups.google.com/group/nodejs/browse_thread/thread/ea64ae28f802e393?fwc=1): second from the last post is on linux. Not sure the fsync method was doing much on stdout anyways. writeSync() does work though and is still preferred over console.log.

**Pull request merged**